src/js/app.js

```diff
@@ -0,0 +1,196 @@
+"use strict";
+
+angular.module('invertedIndex', [])
+  .filter('range', () => {
+    return function (input, number) {
+      for (let i = 1; parseInt(number) >= i; i++) {
+        input.push(i);
+      }
+      return input;
+    };
+  })
+  .directive('fileUpload', ['alerts', (alerts) => {
+    /*
+     * template to be rendered
+     */
+    const template = `<input type="file" id="file" name="files" multiple>
```

Comment:
that is the template for the directive. I understand the point ur trying to make. I will change that to a template URL.